client/src/utils/A-B-tester.test.ts

```diff
@@ -0,0 +1,29 @@
+import faker from 'faker';
+import { emailToAVariant } from './A-B-tester';
+
+describe('client/src is-email-variation-a', () => {
+  it('Consistently returns the same result for the same input', () => {
+    const preSavedResult = {
+      hash: '23e3cacb302b0c759531faa8b414b23709c26029',
+      isAVriant: true,
+      hashInt: 2
+    };
+    const result = emailToAVariant('foo@freecodecamp.org');
+    expect(result).toEqual(preSavedResult);
+  });
+  it('Distributes A and B variations equaly for 100K random emails', () => {
+    let A = 0;
+    let B = 0;
+    const sampleSize = 100000;
+    for (let i = 0; i < sampleSize; i++) {
+      if (emailToAVariant(faker.internet.email()).isAVriant) A++;
```

Comment:
Can we seed faker?  We don't need to keep checking to see if different samples fall close to 50/50.  

I know the chance is low, but I don't think we gain anything by allowing this the possibility of failure.

Reply:
I will look into seeding. If we don't test the 50/50 a/b distribution, what should we test?

Reply:
It's fine to test the distribution, but it should be deterministic.  If we call `faker.seed(123);` or whatever seed you like, then the tests will always generate the same ratio.